src/zeekygen/Manager.cc

```diff
@@ -93,8 +93,7 @@ Manager::Manager(const string& arg_config, const string& command)
 }
 
 Manager::~Manager() {
-    for ( size_t i = 0; i < all_info.size(); ++i )
-        delete all_info[i];
+    std::for_each(all_info.begin(), all_info.end(), [](auto& info) { delete info; });
```

Comment:
Same as above - use a classic for-loop, `std::for_each()` usage increased by >200% with this PR :-)

I'm not against `std::for_each()`, just that we almost never seem to use it, so wonder why we should do it here now.